FogCreek/Glitch-Community
# Superb guide: Get rid of all of the custom ab testing code

## Links
https://superb-guide.glitch.me/
https://app.clubhouse.io/glitch/story/5948/remove-the-custom-ab-testing-code

## Changes:
- Removed all code related to our custom ab testing, since optimizely does the same thing
- Removed the ab testing section from the /secret page

## How To Test:
- Check out the site and make sure it looks unchanged
- Notice the lack of ab tests on the /secret page
